src/components/QuizResults.tsx

```diff
@@ -14,6 +14,8 @@ import TwitterIcon from '@mui/icons-material/Twitter';
 import FacebookIcon from '@mui/icons-material/Facebook';
 import Link from './Link';
 import { HeadWithMeta } from './HeadWithMeta';
+import CircularProgress from '@mui/material/CircularProgress';
+import Backdrop from '@mui/material/Backdrop';
```

Comment:
Is Backdrop being used? (I don't see it in either file)

Reply:
No, ill remove this. something I was just playing around with.